rllib/tuned_examples/dt/pendulum-v1-medium-expert-dt.yaml

```diff
@@ -0,0 +1,48 @@
+pendulum_medium_expert_dt:
+    env: 'Pendulum-v1'
+    run: DT
+    stop:
+        # We could make this -200, but given that we have 4 cpus for our tests, we will have to settle for -300.
+        evaluation/episode_reward_mean: -350
+        timesteps_total: 20000000
+    config:
+        input: 'dataset'
+        input_config:         
+            paths: [
+                'tests/data/pendulum/pendulum_expert_sac_50eps.zip',
+                'tests/data/pendulum/pendulum_medium_sac_50eps.zip',
+            ]
+            format: 'json'
+        num_workers: 3
+        actions_in_input_normalized: True
+        clip_actions: True
+        normalize_actions: True
+        # training
+        framework: torch
+        train_batch_size: 512
+        target_return: -120.0
```

Comment:
Does it not work for other target_returns? say -300? does it reach lower?

Reply:
I found that -120 worked best here. Typically for DT you want to set target_return to be high, either the expert performance or some multiple of the highest in your dataset.